src/FunctionApps/python/requirements.txt

```diff
@@ -4,7 +4,7 @@ azure-functions
 azure-identity
 azure-storage-blob
 hl7
-phdi_transforms @ git+https://github.com/CDCgov/prime-public-health-data-infrastructure@main#subdirectory=src/lib/phdi-transforms
+phdi_building_blocks @ git+https://github.com/CDCgov/prime-public-health-data-infrastructure@dan/TEMP-phdi-building-blocks#subdirectory=src/lib/phdi-building-blocks
```

Comment:
This PR is on `dan/modularization-refactor` and `dan/TEMP-phdi-building-blocks` is a effectively a copy of this branch. Currently the dependency is pointed to this second branch so that when the PR is merged and this branch is deleted nothing (like integration testing) will brake. Planning to follow up with a PR containing a single line change after merging this PR (#109) to point back to main.

Reply:
Here is where @Spencer and I discussed this. https://github.com/CDCgov/prime-public-health-data-infrastructure/pull/109#issuecomment-1127760743